_episodes/38.md

```diff
@@ -0,0 +1,509 @@
+---
+layout: episode
+title:  "38: Trino tacks on polymorphic table functions"
+date: 2022-07-21
+tags: trino polymorphic table functions
+youtube_id: ""
+wistia_id: ""
+sections:
+- title: "Releases 387 to 390"
+  desc: "Details about features and fixes in the new releases"
+  time:
+---
+
+## Guests
+
+In this episode we have the pleasure to chat with a couple familiar faces who
+have been hard at work building and understanding the features we're talking
+about today:
+
+### [Kasia Findeisen](https://github.com/kasiafi), Trino Maintainer
+
+[TODO - Kasia]:<>
```

Comment:
@kasiafi - do you want to write a blurb here about how you're going to introduce yourself, how you've worked on PTF, etc? Only needs to be 1-3 sentences.